_posts/2015-06-07-cosmic-neutrinos.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Improved limits in search for ultrahigh-energy cosmic neutrinos
+author: RaoOfPhysics
+---
+
+_Pierre Auger Observatory finds no neutrino candidates with energies of greater than 1 EeV in data collected between Jan 2004 and Jun 2013_
```

Comment:
"Decade of data collection sharpens neutrino energy measurements"
"Neutrino energy measurement can tell us about cosmic ray origins"
Something along these lines, a little less dry...